v21.2/ui-overview.md

```diff
@@ -30,42 +30,41 @@ Area | Description
+- If you included the [`--http-addr`](cockroach-start.html#networking) flag when starting nodes, use the IP address or hostname and port specified by that flag.
+- If you didn't include the [`--http-addr`](cockroach-start.html#networking) flag when starting nodes, use the IP address or hostname specified by the [`--listen-addr`](cockroach-start.html#networking) flag and port `8080`.
+- If you are running a [secure cluster](#cluster-security), use `https` instead of `http`.
 
-{{site.data.alerts.callout_success}}
 For guidance on accessing the DB Console in the context of cluster deployment, see [Start a Local Cluster](start-a-local-cluster.html) and [Manual Deployment](manual-deployment.html).
-{{site.data.alerts.end}}
 
-### DB Console security
+Access to DB Console is a function of cluster security and the role of the accessing user.
 
-On insecure clusters, all areas of the DB Console are accessible to all users.
+### Cluster security
 
-On secure clusters, for each user who should have access to the DB Console, you must [create a user with a password](create-user.html#create-a-user-with-a-password) and optionally [grant the user membership to the `admin` role](grant.html).
+On insecure clusters, all areas of the DB Console are accessible to all users.
 
-{{site.data.alerts.callout_info}}
-The default `root` user is a member of the `admin` role. Use the following command to [grant users membership to the `admin` role](grant.html):
+On secure clusters, for each user who should have access to the DB Console, you must [create a user with a password](create-user.html#create-a-user-with-a-password) and optionally [`GRANT`](grant.html#grant-role-membership) the user membership to the `admin` role.
 
-<code style="white-space:pre-wrap">GRANT admin TO \<sql_user\>;</code>
-{{site.data.alerts.end}}
+### Role-based security
 
-For security reasons, non-admin users access only the data over which they have privileges (e.g., their tables, jobs, and list of sessions), and data that does not require privileges (e.g., cluster health, node status, metrics).
+All users have access to data over which they have privileges (e.g., [jobs](ui-jobs-page.html) and [list of sessions](ui-sessions-page.html)), and data that does not require privileges (e.g., [cluster health, node status](ui-cluster-overview-page.html), [metrics](ui-overview-dashboard.html)).
 
-The following areas of the DB Console can be accessed only by [`admin` users](authorization.html#admin-role). These areas display information from privileged HTTP endpoints that operate with `admin` privilege.
+[`admin` users](authorization.html#admin-role) also have access to the following areas. These area display information from privileged HTTP endpoints that operate with `admin` privilege.
 
-Secure area | Privileged information
+DB Console areas | Privileged information
```

Comment:
Should this be singular "DB Console area" ?

Reply:
yes, thanks for the catch. fixed.